known_keypoints:
  - "red_button"
  - "left_button"
  - "red_plug"
  - "black_plug"
  - "screw_1"
  - "screw_2"
  - "screw_3"
  - "screw_4"
  - "screw_5"
  - "screw_6"
  - "screw_7"
  - "screw_8"
  - "cover_screw_top_left"
  - "cover_screw_top_center"
  - "cover_screw_top_right"
  - "cover_screw_bottom_left"
  - "cover_screw_bottom_center"
  - "cover_screw_bottom_right"
  - "screen_corner_left_1"
  - "screen_corner_left_2"
  - "screen_corner_left_3"
  - "screen_corner_left_4"
  - "screen_corner_left_4"
  - "cable_wrap_screw_left_1"
  - "cable_wrap_screw_left_2"
  - "cable_wrap_screw_left_3"
  - "cable_wrap_screw_left_4"
  - "cable_wrap_screw_right_1"
  - "cable_wrap_screw_right_2"
  - "cable_wrap_screw_right_3"
  - "cable_wrap_screw_right_4"
  - "door_screw_left"
  - "door_screw_right"
  - "door_screw_left"
  - "door_screw_right"
  - "slider"
